windows/nearobject/service/NearObjectDeviceDiscoveryAgentUwb.cxx

```diff
@@ -23,6 +24,30 @@ CreateNearObjectUwbDevice(std::string deviceName)
 }
 } // namespace detail
 
+NearObjectDeviceDiscoveryAgentUwb::NearObjectDeviceDiscoveryAgentUwb() :
+    m_devicePresenceMonitor(
+        windows::devices::uwb::InterfaceClassUwb, [this](auto &&deviceGuid, auto &&presenceEvent, auto &&deviceName) {
```

Comment:
For debugging convenience, it'd be nice if the lambda just called a member function with the same signature as the callback and forwarded all the arguments. I sent PR #260 to merge to your topic branch with the change.